client/src/pages/dashboard/components/navbar.jsx

```diff
@@ -5,15 +5,22 @@ import Image from 'next/image'
 // IMPORT ICONS
 import ucsdLogo from '../../../assets/images/ucsdLogo.png';
 
+// IMPORT UTILS
+import { logout } from '../actions/logout_actions';
+
 const Navbar = () => {
   return (
     <nav className="bg-white">
         <div className="container mx-auto px-4 py-2 flex justify-between items-center">
             <h1 className="font-bold text-2xl text-black">STUDENT HEALTH SERVICES</h1>
             <Image src={ucsdLogo} alt="Logo" className="w-auto h-6" />
         </div>
-        
         <hr className="border-t border-[#6DE4EA] mx-12" />
+        <div className="flex justify-end">
```

Comment:
The position of Logout is slightly off, is it only in my screen?

<img width="1499" alt="Screen Shot 2024-02-29 at 1 07 32 PM" src="https://github.com/rohithaug/UCSD-CSE-210-Team-12/assets/64318022/c532e5c9-5e8e-4df1-8f4c-cb317967c5cf">

Reply:
I placed it beneath the `<hr>` tag. But it doesn't appear this much off on my screen.

![image](https://github.com/rohithaug/UCSD-CSE-210-Team-12/assets/59926447/45da9285-93fb-45fd-bb8a-2a08b2636b05)

Reply:
May be add a little bit of margin to the right. Reference: https://tailwindcss.com/docs/margin